pkg/activitypub/client/transport/transport.go

```diff
@@ -20,9 +20,14 @@ import (
 var logger = log.New("activitypub_client")
 
 const (
-	contentTypeHeader          = "Content-Type"
-	acceptHeader               = "Accept"
-	activityStreamsContentType = `application/ld+json; profile="https://www.w3.org/ns/activitystreams"`
+	// AcceptHeader specifies the content type that the client is expecting.
+	AcceptHeader = "Accept"
+
+	// LdPlusJSONContentType specifies the linked data plus JSON content type.
+	LdPlusJSONContentType = `application/ld+json`
```

Comment:
Should technically be `LDPlusJSONContentType`

Reply:
I'll update it next time I push.